Documentation/Signoffs/Mounting.md

```diff
@@ -0,0 +1,123 @@
+
+The mounting subsystem should be designed to ensure that the weighted average noise does not exceed 85 dB from the overall system to guarantee no hearing loss will occur.
+
+#### 8.) System shall follow OSHA standard 1910.304-305 for wiring design and protection. [Origin: OSHA standards]
+
+The mounting subsystem should be designed such that wires are easily identified and discerned from other conductors, for example the ground wire must be easily found when comparing it to live conductors. The wires must also be insulated and be in a proper location to minimize the probability of electrical shock occuring.
+
+#### 9.) System shall follow the IEEE 518-1982 Guide for the Installation of Electrical Equipment to Minimize Electrical Noise Inputs to Controllers from External Sources. [Origin: IEEE standards]
+
+The mounting subsystem needs to follow this guide to limit outside noise which could lower the effectiveness of the noise cancellation capability of the overall system.
+
+#### 10.) Overall weight of mounting subsystem cannot exceed 17 lbs. [Origin: Weight requirement ]
+
+The mounting subsystem cannot be physically drilled into the wall since this would damage Tennessee Tech property, therefore command strips will be utilized, these command strips are rated for 20 lbs. The system will add weight, therefore to compensate for the additional weight, the weight requirement will be lessened to a maximum amount of 17 lbs for the physical mount to prevent failure.
+#### 11.) The width of the physical mount cannot exceed 49 in. [Origin: Window width measurements ]
+
+The physical mount cannot be wider than the window in order to be fastened to the window. The measured width of the window in Brown 314 is 49 inches (4 feet 1 inch), therefore the selected physical mount must meet this requirement.
+
+
+
+
+
+
+## Buildable schematic 
+<img src="/Documentation/Images/Mounting/Top_Down.png" width= "80%" height = "80%" alt=" Buildable Schematic of mounting system top-down view">
+
+*Figure 1. Buildable Schematic of mounting system top-down view*
+
+
+<img src="/Documentation/Images/Mounting/Side_View.png" alt=" Buildable Schematic of mounting system side view">
+
+*Figure 2. Buildable Schematic of mounting system side view*
+
+
+The third section should show the buildable schematic directly embedded in the markdown file as a jpeg image. If the schematic is not clearly readable and appropriately sized, the supervisor will reject the signoff. 
+
+The schematic must be appropriate to the design. ie. 3d model for a physical system or wiring schematic for a circuit. Further, the schematic(s) must contain every detail necessary for the design to be built by someone who has no knowledge of the design. Every relevant component value and measurement must be given.
+
+The actual design artifacts (3d model, Cad files, gerber files, etc) must be uploaded to the documentation folder in the appropriate location.
+
+## Analysis
+
+1.) Physical mount details
```

Comment:
This does not seem appropriate. Why not 3d print an enclosure that is more appropriate for free instead of spending 50 dollars and it looking less professional?